src/NServiceBus.Core/Forwarding/ForwardingToDispatchConnector.cs

```diff
@@ -0,0 +1,16 @@
+namespace NServiceBus
+{
+    using System;
+    using NServiceBus.Forwarding;
+    using NServiceBus.Pipeline;
+    using NServiceBus.TransportDispatch;
+    using NServiceBus.Transports;
+
+    class ForwardingToDispatchConnector:StageConnector<ForwardingContext,DispatchContext>
+    {
+        public override void Invoke(ForwardingContext context, Action<DispatchContext> next)
+        {
+            next(new DispatchContext(context.Get<OutgoingMessage>(),context));
```

Comment:
Some (most) connectors don't fork, they connect stages 1:1 so we need to figure out a name for them. I am not an expert on hydraulics-related terminology, especially in English, but I am :+1: for having proper names.